Source/AppDelegate.m

```diff
@@ -1,5 +1,5 @@
 #import "AppDelegate.h"
-#import "IPFTestRunnerViewController.h"
+#import "iperf-Swift.h"
```

Comment:
Again here, if we go Swift we should probably go full Swift rather than a mix of both languages, and calling headers `-Swift` feels a bit off in that world

Reply:
Switched to a Swift AppDelegate, so no more import of -Swift.h file